server/appointment/upload-documents/uploadDocuments.php

```diff
@@ -197,10 +211,50 @@ function markAppointmentAsReady($token, $firstName, $lastName, $emailAddress, $p
 	echo json_encode($response);
 }
 
+function storeConsent($reviewConsent, $virtualConsent, $signature, $appointmentId) {
+	GLOBAL $DB_CONN;
+
+	$response = array();
+	$response['success'] = false;
+	
+	if($virtualConsent === 'true' && $virtualConsent !== null && $signature !== null && $signature !== '' && $appointmentId !== null) {
```

Comment:
I would do `$virtualConsent != null && $virtualConsent === 'true' && $signature != null && trim($signature) !== '' && $appointmentId != null`. 

Just using `!=` for the null checks, and trimming the signature

Reply:
I saw elsewhere on the page, null was checked with ===, not ==. How do you know when to use which?

Reply:
So, I think when we want to check type+value, we should use `===`, and when we just care about value, we should use `==`. Technically, I think it's tomato/tomato in the case of `null`, but I like the idea of `==` better.. In JS for example, it's better to use `==` when checking for `null` or `undefined` b/c it will match either of them (or something like this, can't rightly remember). 

There's actually also a `is_null($val)` function in PHP that could be used instead too, but I think `==` is fine. Doesn't necessarily need a change.

Reply:
Yeah, you're right about the == checking for null or undefined. Changed this!